docs/knowledge_base/threat_feed.md

```diff
@@ -8,15 +8,15 @@ The Phylum threat feed provides a curated view into malware being released acros
 
 1. Obtain an [API key](../knowledge_base/api-keys.md) and set it as follows:
 
-    ```bash
-    PHYLUM_API=p0_...
-    ```
+   ```bash
+   PHYLUM_API=p0_...
+   ```
 
 2. Use your API key to retrieve the latest packages in the threat feed:
 
-    ```bash
-    curl https://threats.phylum.io -H "Authorization: Bearer $PHYLUM_API"
-    ```
+   ```bash
+   curl https://threats.phylum.io -H "Authorization: Bearer $PHYLUM_API"
```

Comment:
who does have access to the thread feed?

Reply:
```{"code":401,"description":"You do not have access to the threat feed!","name":"error"}

Reply:
@louislang would just have to add you to the authorized list.

Reply:
aha! i shall request :-)